flexget/plugins/api_t411.py

```diff
@@ -311,7 +311,7 @@ def search(self, query):
             url_params['offset'] = query['page_index']
         if query.get('terms') is not None:
             for (term_type_id, term_id) in query['terms']:
-                term_type_key_param = 'term[' + str(term_type_id) + '][]'
+                term_type_key_param = u"term[%s][]" % str(term_type_id)
```

Comment:
if you check the issue you would see that it absolutely needs to be a unicode string. (or everything else needs to be binary string.

As for the str(). I'm not a python expert. But the referenced site said that it needs the str().

@liiight at the top of the file theres `from builtins import * `. So I don't think it's necessary